frontend/homework/9/src/profile/Skills.css

```diff
@@ -0,0 +1,27 @@
+.skills-container {
+    margin-top: 20px;
```

Comment:
All the colors, font related constants must be stored in the separate constants file